src/toil/batchSystems/parasolTestSupport.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import absolute_import
+from abc import ABCMeta, abstractmethod
+import logging
+import shutil
+import threading
+import time
+import subprocess
+import multiprocessing
+import os
+from toil.lib.bioio import getTempFile
+
+log = logging.getLogger(__name__)
+
+
+class ParasolTestSupport(object):
+    """
+    For test cases that need a running Parasol leader and worker on the local host
+    """
+    def _startParasol(self, numCores=None):
+        if numCores is None:
+            numCores = multiprocessing.cpu_count()
+        #shutil.rmtree('/tmp/parasol', ignore_errors=True)
```

Comment:
We try to avoid code that is commented out. Ask me to explain why. 

Either remove completely or make it conditional.